worlds/sc2/MissionTables.py

```diff
@@ -734,4 +756,4 @@ def get_campaign_potential_goal_missions(campaign: SC2Campaign) -> List[SC2Missi
 
 
 def get_no_build_missions() -> List[SC2Mission]:
-    return [mission for mission in SC2Mission if not mission.build]
+    return [mission for mission in SC2Mission if not mission.flags & MissionFlags.NoBuild]
```

Comment:
```suggestion
    return [mission for mission in SC2Mission if mission.flags & MissionFlags.NoBuild]
```
Doing some quick mental bitwise math, this should probably be flipped. Feel free to tell me I'm wrong if you tested this.